roles/cdh/templates/base.j2

```diff
@@ -16,9 +16,14 @@
     {% set serv_j = joiner(",") %}
     {% for service in (cdh_services|selectattr('type', 'in', cdh_service_types | default(cdh_services | map(attribute='type'))))  %}
     {{ serv_j() }}
-    {% set t = service['type'] + '.j2' %}{% include t %}
+    {% if service['type'] + '_def_override' in vars %}
```

Comment:
Do `***_def_override` vars also include predefined configurations when we define it? For example, `hdfs_def_override` vars may include some configurations described in `hdfs.j2` if necessary?